deployment/network-operator/templates/tests/test-dev-plugin.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and .Values.deployCR .Values.devicePlugin.deploy  }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-dev-plugin-test"
+  annotations:
+    helm.sh/hook: test-success
```

Comment:
we only support helm v3, you can use updated test hook annotation

https://helm.sh/docs/topics/chart_tests/

Reply:
changed to test